inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/utils/MessageEventReconciliationUtil.kt

```diff
@@ -259,17 +259,19 @@ internal interface MessageEventReconciliationUtil {
         @SuppressWarnings("FunctionMaxLength")
         private fun getNumTimesToSatisfyTriggersForDisplay(message: Message): Int {
             val maxImpression = message.getMaxImpressions()
-            val displayedImpression: Int = LocalDisplayedMessageRepository.instance().numberOfTimesDisplayed(message)
+            val displayedImpression: Int = LocalDisplayedMessageRepository.instance()
+                    .numberOfTimesDisplayed(message)
+            val displayedImpressionAfterLastPing: Int = LocalDisplayedMessageRepository.instance()
+                    .numberOfDisplaysAfterPing(message)
             val incrementRemoved = message.getCampaignId()?.let {
```

Comment:
Please consider this case:
* campaign is displayed
* kill app (without closing the campaign)
* on app restart, the campaign should not be displayed

This is handled using [incrementRemoved PR](https://github.com/rakutentech/android-inappmessaging/pull/84).
This should be handled correctly.

Reply:
I tested this scenario and it's working fine, the campaign is not displayed after app restart.

Reply:
then, was it displayed after triggering the events again (once)?

Reply:
it is an issue when I tested it.
you need to trigger the event twice before it is displayed, this is because of the additional counter.
since we are clearing old events now, it is better to remove this counter.
I think same goes with `close message`